### PR TITLE
📦 NEW: IR for Blaupunkt and Grundig TV

### PR DESCRIPTION
**Pull request by nnesetto**

### CODE REVIEW

File: TVs/Blaupunkt/Blaupunkt.ir [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+address: 00 7F 00 00
+command: 0A F5 00 00
+# 
+name: Mute
+type: parsed
+protocol: NECext
+address: 00 7F 00 00
+command: 50 AF 00 00
+# 
+name: Vol_up
```
Comment (darmiel) [inline TVs/Blaupunkt/Blaupunkt.ir]:
```suggestion
name: VOL+
```

File: TVs/Blaupunkt/Blaupunkt.ir [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+type: parsed
+protocol: NECext
+address: 00 7F 00 00
+command: 1E E1 00 00
+# 
+name: Vol_down
```
Comment (darmiel) [inline TVs/Blaupunkt/Blaupunkt.ir]:
```suggestion
name: VOL-
```

File: TVs/Blaupunkt/Blaupunkt.ir [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+address: 00 7F 00 00
+command: 5F A0 00 00
+# 
+name: Ch_up
+type: parsed
+protocol: NECext
+address: 00 7F 00 00
+command: 1F E0 00 00
+# 
+name: Ch_down
```
Comment (darmiel) [inline TVs/Blaupunkt/Blaupunkt.ir]:
```suggestion
name: CH-
```

File: TVs/Blaupunkt/Blaupunkt.ir [hunk 4]
```diff
@@ -0,0 +1,122 @@
+Filetype: IR signals file
+Version: 1
+# 
+name: Power
```
Comment (darmiel) [inline TVs/Blaupunkt/Blaupunkt.ir]:
```suggestion
name: POWER
```

File: TVs/Grundig/Grundig_2.ir [hunk 5]
```diff
@@ -0,0 +1,110 @@
+Filetype: IR signals file
+Version: 1
+# 
+name: Power
```
Comment (darmiel) [inline TVs/Grundig/Grundig_2.ir]:
```suggestion
name: POWER
```

File: TVs/Grundig/Grundig_2.ir [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+duty_cycle: 0.330000
+data: 974 718 1823 715 947 746 921 798 893 773 919 798 894 802 893 1644 1744 795 893 1644 918 774 918 85309 893 799 1768 770 918 775 916 777 914 778 914 779 913 783 913 1625 1764 776 913 1625 913 780 913
+# 
+name: Right
+type: raw
+frequency: 38000
+duty_cycle: 0.330000
+data: 973 718 974 717 1825 715 972 718 972 721 919 798 894 802 893 1644 1743 794 893 1643 1769 86120 893 798 894 798 1769 770 917 776 915 777 914 778 914 781 914 1624 1765 774 914 1624 1765
+# 
+name: Vol_up
```
Comment (darmiel) [inline TVs/Grundig/Grundig_2.ir]:
```suggestion
name: VOL+
```

File: TVs/Grundig/Grundig_2.ir [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+type: raw
+frequency: 38000
+duty_cycle: 0.330000
+data: 972 719 1823 716 973 719 972 720 945 747 920 772 920 802 893 1645 1743 796 918 773 918 774 916 86148 917 774 1767 772 916 777 914 778 913 779 913 779 913 783 913 1625 1763 776 913 779 913 779 913 86148 918 773 1767 773 915 778 914 779 913 779 913 780 912 783 913 1625 1764 777 912 780 912 780 912
+# 
+name: Vol_down
```
Comment (darmiel) [inline TVs/Grundig/Grundig_2.ir]:
```suggestion
name: VOL-
```

File: TVs/Grundig/Grundig_2.ir [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+type: raw
+frequency: 38000
+duty_cycle: 0.330000
+data: 946 745 947 745 1850 688 945 746 944 748 893 798 893 802 893 1644 1743 794 894 797 894 1643 918 85284 892 798 894 798 1769 769 918 775 915 778 914 778 914 782 913 1649 1739 775 913 778 914 1625 913 85274 943 771 919 773 1766 773 915 778 914 777 915 777 915 781 914 1623 1765 774 914 778 914 1623 914
+# 
+name: Ch_up
```
Comment (darmiel) [inline TVs/Grundig/Grundig_2.ir]:
```suggestion
name: CH+
```

File: TVs/Grundig/Grundig_2.ir [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+type: raw
+frequency: 38000
+duty_cycle: 0.330000
+data: 972 719 1822 715 972 717 973 719 970 721 919 772 919 1647 1743 794 894 797 894 797 918 773 918 86132 892 798 1768 770 918 774 916 778 913 803 889 804 888 1653 1739 800 889 804 888 804 888 804 888 86144 918 773 1769 771 915 777 915 777 915 778 914 778 914 1627 1765 774 914 777 915 777 915 778 914
+# 
+name: Ch_down
```
Comment (darmiel) [inline TVs/Grundig/Grundig_2.ir]:
```suggestion
name: CH-
```

File: .gitignore
Comment (darmiel) [inline .gitignore:1]: Please remove this file from the PR

**Review by darmiel — CHANGES_REQUESTED** [review]
Looks good so far, you already fixed the naming issues, but please remove the .gitignore file :)

File: TVs/Blaupunkt/Blaupunkt.ir [hunk 10]
```diff
@@ -0,0 +1,122 @@
+Filetype: IR signals file
+Version: 1
+#
```
Comment (darmiel) [inline TVs/Blaupunkt/Blaupunkt.ir:3]: Can you add a comment with the TV model here?

```suggestion
# 
# Blaupunkt XYZ
#
```

File: TVs/Grundig/Grundig_2.ir [hunk 11]
```diff
@@ -0,0 +1,110 @@
+Filetype: IR signals file
+Version: 1
+#
```
Comment (darmiel) [inline TVs/Grundig/Grundig_2.ir:3]:
```suggestion
# 
# Here aswell :)
#
```

**Comment by nnesetto** [comment]
I restored the state of .gitignore to what it was in the original repository.

Regarding TV models, unfortunately, I do not know what models they were because they were in the hotel.

**Comment by darmiel** [comment]
> I restored the state of .gitignore to what it was in the original repository.
> 
> Regarding TV models, unfortunately, I do not know what models they were because they were in the hotel.

Alright, that's fine. Thank you very much 🙂

**Pull request merged**